pkgs/top-level/all-packages.nix

```diff
@@ -5094,6 +5094,8 @@ in
 
   intercal = callPackage ../development/compilers/intercal { };
 
+  irony-server = callPackage ../development/tools/irony-server/default.nix { };
```

Comment:
I think it's better to pass in `irony` as an argument here instead of resolving it in the expression, eg.

```
irony-server = callPackage ../development/tools/irony-server/default.nix {
  inherit (emacsPackagesNg.melpaStablePackages) irony;
};
```